docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: "3.5"
+
+services:
+  postgresql:
+    image: postgres:9.6
```

Comment:
you can't use a more recent image ?

Reply:
Sure I can. I choose this version used in other FUN projects. What version do you suggest?

Reply:
The more recent grafana can use ?

Reply:
Okay I've switched to `13.3`